src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -356,23 +356,23 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
-        {"InternalFeatureOnOffOptions_OOP64Bit", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOP64Bit")},
-        {"InternalFeatureOnOffOptions_OOPCoreClrFeatureFlag", new FeatureFlagStorage(@"Roslyn.ServiceHubCore")},
-        {"InternalFeatureOnOffOptions_OOPServerGCFeatureFlag", new FeatureFlagStorage(@"Roslyn.OOPServerGC")},
-        {"InternalFeatureOnOffOptions_RemoveRecommendationLimit", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "RemoveRecommendationLimit")},
-        {"InternalFeatureOnOffOptions_RenameTracking", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Rename Tracking")},
-        {"InternalFeatureOnOffOptions_SemanticColorizer", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Semantic Colorizer")},
-        {"InternalFeatureOnOffOptions_ShowDebugInfo", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "ShowDebugInfo")},
-        {"InternalFeatureOnOffOptions_SmartIndenter", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Smart Indenter")},
-        {"InternalFeatureOnOffOptions_Snippets", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Snippets2")},
-        {"InternalFeatureOnOffOptions_Squiggles", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Squiggles")},
-        {"InternalFeatureOnOffOptions_SyntacticColorizer", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Syntactic Colorizer")},
+        {"dotnet_internal_feature_on_off_options_automatic_line_ender", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Automatic Line Ender")},
+        {"dotnet_internal_feature_on_off_options_brace_matching", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Brace Matching")},
+        {"dotnet_internal_feature_on_off_options_classification", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Classification")},
+        {"csharp_internal_feature_on_off_options_event_hook_up", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Event Hookup")},
+        {"visual_basic_internal_feature_on_off_options_format_on_save", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "FormatOnSave")},
+        {"dotnet_internal_feature_on_off_options_full_solution_analysis_memory_monitor", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Full Solution Analysis Memory Monitor")},
+        {"dotnet_internal_feature_on_off_options_oop_64_bit", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOP64Bit")},
+        {"dotnet_internal_feature_on_off_options_oop_core_clr_feature_flag", new FeatureFlagStorage(@"Roslyn.ServiceHubCore")},
+        {"dotnet_internal_feature_on_off_options_OOP_server_gc_feature_flag", new FeatureFlagStorage(@"Roslyn.OOPServerGC")},
```

Comment:
Unintentional capitalization of `OOP`?

Reply:
Ah good catch

Reply:
I will go via all the strings again when my brain is clear later  : )